util/env/env.go

```diff
@@ -133,13 +131,12 @@ func ParseDurationFromEnv(env string, defaultValue, min, max time.Duration) time
 	if str == "" {
 		return defaultValue
 	}
-	durPtr, err := timeutil.ParseDuration(str)
+	dur, err := time.ParseDuration(str)
```

Comment:
@mpelekh what was the reason for changing this library? Someone [pointed out](https://github.com/argoproj/argo-cd/issues/24740) that it is a breaking change.

Reply:
From what I can see it was to support durations in ms, old library doesn't: https://github.com/argoproj/pkg/blob/master/time/time.go#L31

new one does: https://pkg.go.dev/time#ParseDuration

Reply:
Exactly. The reason for switching to `time.ParseDuration` was to support durations in milliseconds (the default value for `clusterCacheEventProcessingInterval` is 100ms), because the [utility function](https://github.com/argoproj/pkg/blob/master/time/time.go) doesn't support that.